### PR TITLE
setup project configuration

### PR DESCRIPTION
**Pull request by sebas962**
Initial configuration of the repository

### CODE REVIEW

File: .github/package.json [hunk 1]
```diff
@@ -14,4 +18,6 @@
   ],
   "author": "Sebastian Gomez Graciano <sebastian.gomezg@perficient.com> (https://github.com/sebas962)",
   "license": "MIT"
+
+
```
Comment (manuelq12) [inline .github/package.json]: you can erase these spaces

Comment (sebas962) [reply .github/package.json]: spaces deleted

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+ @david31982 @holgiosalos @JuanDelahozMiranda @manuelq12 @valeryibarra
```
Comment (manuelq12) [inline .github/CODEOWNERS]: EOF, we require a line break here

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+
```
Comment (manuelq12) [inline .github/CODEOWNERS:2]: It was only one, but is okay

**Review by manuelq12 — DISMISSED** [review]

File: .github/package.json [hunk 4]
```diff
@@ -1,8 +1,20 @@
 {
+  "name": "cypress-training",
+  "version": "1.0.0",
+  "description": "This is Workshop about Cypress",
+  "main": "index.js",
   "scripts": {
-    "test": "cypress run"
+    "test": "cypress open"
```
Comment (manuelq12) [inline .github/package.json]: test:open not test

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**